app/views/tutorials/new.html.erb

```diff
@@ -13,6 +13,15 @@
         <%= f.label :tagline %>
         <%= f.text_field :tagline, class: 'form-control' %>
 
+        <%= f.label :author %>
+        <%= f.text_field :author, class: 'form-control' %>
```

Comment:
If only registered users can create tutorials, shouldn't we read from the User object instead of having the user type in a name?